test-cases/longevity/longevity-topology-changes-3h.yaml

```diff
@@ -22,6 +23,7 @@ nemesis_class_name: 'SisyphusMonkey'
 nemesis_selector: ['topology_changes']
 nemesis_interval: 5
 nemesis_filter_seeds: false
+nemesis_add_node_cnt: 2
```

Comment:
It's a weird case to test RF=3 with 2 AZs and adding 2 nodes.
I don't even know what we should expect.
For now, let's go with the recommended and supported case: RF=3, 3 AZs, adding 3 nodes every time.

Reply:
Also, let's change:
n_db_nodes to 3
seeds_num: 3

@aleksbykov please review these suggested changes as well.